src/main/app/pdf/mappers/theirDetailsMapper.ts

```diff
@@ -43,6 +44,28 @@ export class TheirDetailsMapper {
     }
   }
 
+  private static splitNames (partyDetails: TheirDetails): object {
+    if (partyDetails && partyDetails.type) {
+      switch (partyDetails.type) {
```

Comment:
is there any benefit to having this switch? you're returning the base partyDetails properties so you could just cast as PartyDetails or just as Individual and return the same title, firstName, lastName. there's nothing specific to Individual or SoleTrader that is being sent

Reply:
you're right. we have a similar switch in all of the methods of this class (even when there's only one clause) and I was being foolishly consistent :)